Use Cases/SDPi/Temp from Repo/tf1-ch-c-use-case-stad.adoc

```diff
@@ -0,0 +1,97 @@
+<<<<<<< HEAD
```

Comment:
I guess I've found a culprit here...

`Use Cases/SDPi/Temp from Repo/tf1-ch-c-use-case-stad.adoc`

What path is that? "Temp from Repo" looks suspicious. Is this folder really suppose to exists and be merged to master?

Reply:
Don't have a clue ... the REAL file is in SDPi_Supplement/asciidoc/volume1/use-cases.

That said, for my work I've been using:  github browser tools, Github Desktop, IntelliJ terminal for git command line.

Ken has been using the same EXCEPT he doesn't use the Github Desktop but uses the IntelliJ git interface for all VCS stuff. I HATE that IntelliJ git interface ... totally confusing, never clear what to do, where to do, what DID it do, etc.  

Question:  Perhaps in Ken's using that, my using my stuff, etc.  that problem I resolved last week ended up in this totally bogus "resolve conflicts" file getting included in the mix and mucking stuff up?  IOW perhaps the original problem I had was rooted in the IntelliJ git interface - which is the only unique element of the tool chain.

And what you are experiencing was not caused by David but by this funky took interaction last week that I was able to resolve in my local-remote-branch-PR world but left residue elsewhere ... like in your PR #167 world.  ????!!!!!!!

Reply:
So I conclude: people were confused and this folder should be deleted, right? :-)

Reply:
Directory deleted ... !!!